AAEmu.Game/Models/Game/Char/Actability.cs

```diff
@@ -9,9 +10,50 @@ public class Actability
     public int Point { get; set; }
     public byte Step { get; set; }
 
+    // These are 3.x values, and might not be correct for 1.2
+    private static readonly float[] s_expMultipliers       = [1.00f, 1.20f, 1.40f, 1.60f, 1.80f, 2.00f, 2.20f, 2.40f, 2.60f, 2.80f, 3.00f, 3.30f];
+    private static readonly float[] s_laborCostMultipliers = [1.00f, 1.00f, 0.95f, 0.90f, 0.85f, 0.80f, 0.80f, 0.80f, 0.80f, 0.75f, 0.70f, 0.60f];
+    private static readonly float[] s_timeMultipliers      = [1.00f, 0.97f, 0.94f, 0.94f, 0.94f, 0.88f, 0.88f, 0.88f, 0.84f, 0.84f, 0.80f, 0.74f];
+
     public Actability(ActabilityTemplate template)
     {
         Id = template.Id;
         Template = template;
     }
+
+    /// <summary>
+    /// Gets Exp multiplier for the current skill level
+    /// </summary>
+    /// <returns></returns>
+    public float GetExpMultiplier()
+    {
+        return s_expMultipliers[Math.Clamp(Step, 0, s_expMultipliers.Length - 1)];
+    }
```

Comment:
One liner returns can be converted to `=>`. This and others.

```suggestion
    public float GetExpMultiplier()
        => s_expMultipliers[Math.Clamp(Step, 0, s_expMultipliers.Length - 1)];
```

Reply:
Will likely update this later when I'm actually cleaning this up.
Current PR is mostly so I can make people test it.
But duly noted 😸